src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -183,7 +183,7 @@ const DiscoverySearch = ({
       <View style={styles.searchContainer}>
         <View style={styles.wrapperSearch}>
           <View style={styles.wrapperIcon}>
-            <MemoIcSearch width={16.67} height={16.67} />
+            <MemoIcSearch width={16.67} height={16.67} fill={COLORS.gray300} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color of the `MemoIcSearch` component has been added and set to gray. This should be fine as long as it fits with the overall design and doesn't affect the visibility of the icon.
```diff
- <MemoIcSearch width={16.67} height={16.67} />
+ <MemoIcSearch width={16.67} height={16.67} fill={COLORS.gray300} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->